pkgs/development/python-modules/beautiful-date/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, python-dateutil
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "beautiful-date";
+  version = "2.2.0";
+  format = "setuptools";
+
+  disable = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "kuzmoyev";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-5xRmHaAPf1ps75cOINHkHT1aYb5UGLZGl0OHVQaMES0=";
+  };
+
+  propagatedBuildInputs = [
+    python-dateutil
+  ];
+
+  checkInputs = [
```

```suggestion
  nativeCheckInputs = [
```
Please use nativeCheckInputs instead.